modules/extensions-support/src/main/java/org/mule/module/extension/internal/manager/DefaultExtensionManager.java

```diff
@@ -262,6 +294,41 @@ private void logExtensionHotUpdate(Extension extension, Extension actual)
         }
     }
 
+    private ConfigurationExpirationMonitor newConfigurationExpirationManager()
```

Comment:
this is not a dependency, this is a factory method. Also notice that the returned monitor requires this mananger's extension registry and has a BiConsumer which executes private methods of this manager. I don't think this is a dependency you can inject